pages/tutorials/github_actions.md.erb

```diff
@@ -0,0 +1,82 @@
+
+{:toc}
+
+The Buildkite GitHub Action allows you to:
+
+* Create builds in Buildkite pipelines and set `commit`, `branch`, `message`.
+* Save the build JSON response to `${HOME}/${GITHUB_ACTION}.json` for downstream actions.
+
+<div class="Docs__note">
+<h3 class="Docs__note__heading">Prerequisites</h3>
+<p>This tutorial assumes some familiarity with GitHub and using GitHub Actions. You can find the official GitHub Actions documentation <a href="https://docs.github.com/en/actions/learn-github-actions">here</a>.</p>
+</div>
+
+## Installing the Buildkite GitHub Action
+
+1. Search for Trigger Buildkite Pipeline on [GitHub Marketplace](https://github.com/marketplace) or follow [this link](https://github.com/marketplace/actions/trigger-buildkite-pipeline).
+
+2. On the [Trigger Buildkite Pipeline](https://github.com/marketplace/actions/trigger-buildkite-pipeline) page, click "Use latest version".
+
+3. Copy and paste the following snippet into your .yml file.
+
+```yml
+- name: Trigger Buildkite Pipeline
+  uses: buildkite/trigger-pipeline-action@v1.2.0
+```
+{: codeblock-file="pipeline.yml"}
+
+<%= image "trigger-buildkite-pipeline.png", width: 2630/2, height: 1692/2, alt: "Trigger Buildkite Pipeline GitHub Action on GitHub" %>
+
+## Creating and configuring the workflow for Buildkite GitHub Actions
+
+1. Create a [Buildkite API access token](/docs/apis/rest-api#authentication) with `write_builds` [scope](/docs/apis/managing-api-tokens#token-scopes), and save it to your GitHub repository’s Settings in Secrets.
+
+2. Configure your GitHub Actions workflow with the details of the pipeline to be triggered, and the settings for the build.
+
+For example, the following workflow creates a new Buildkite build on every commit (change `my-org` to the slug of your org):
+
+```
+workflow "Trigger a Buildkite Build" {
+  on = "push"
+  resolves = ["Build"]
+}
+
+action "Build" {
+  uses = "buildkite/trigger-pipeline-action@v1.2.0"
+  secrets = ["BUILDKITE_API_ACCESS_TOKEN"]
+  env = {
+    PIPELINE = "my-org/my-deploy-pipeline"
+    COMMIT = "HEAD"
+    BRANCH = "master"
+    MESSAGE = ":github: Triggered from a GitHub Action"
+  }
+}
+```
+
+## Configuring the workflow
+
+Use the following configuration options:
+
+|Env var|Description|Default|
+|-|-|-|
+|PIPELINE|The pipeline to create a build on, in the format `<org-slug>/<pipeline-slug>`||
+|COMMIT|The commit SHA of the build. Optional.|`$GITHUB_SHA`|
+|BRANCH|The branch of the build. Optional.|`$GITHUB_REF`|
+|MESSAGE|The message for the build. Optional.||
+|BUILD_ENV_VARS|Additional environment variables to set on the build, in JSON format. For example, `{"FOO": "bar"}`. Optional. ||
+
+## Testing for development
+
+To run the test workflow locally, you can use [act](https://github.com/nektos/act, which will run it the same way it does on GitHub:
+
+```
+act
+```
+
+## Releasing
+Create a new GitHub release. The version numbers in the readme will be automatically updated.
```

Comment:
This is probably not meant to be here?

Reply:
Now that I think about it, the contributing section is definitely overkill in a tutorial :D